data/src/main/java/me/androidbox/data/remote/model/response/AttendeeDto.kt

```diff
@@ -4,7 +4,7 @@ data class AttendeeDto(
     val email: String,
     val fullName: String,
     val userId: String,
-  //  val eventId: String,
-  //  val isGoing: Boolean,
-  //  val remindAt: Long
+    val eventId: String = "",
+    val isGoing: Boolean = false,
+    val remindAt: Long = 0L
```

Comment:
Why the default values?

Reply:
@MKiperszmid The default values was because the endpoint doesn't send them so they are optional.